java/com/google/openlocationcode/OpenLocationCode.java

```diff
@@ -153,11 +154,12 @@ public double getEastLongitude() {
    * @constructor
   */
   public OpenLocationCode(String code) throws IllegalArgumentException {
-    if (!isValidCode(code.toUpperCase())) {
```

Comment:
We can get rid of the code.toUpperCase() here all together, since we're calling it inside of isValidCode().

Reply:
since this is public, can we make that assumption and document it the constructor comments?

Reply:
I don't understand your question here, can you rephrase? What assumption are you thinking we need to document?